<launch>   
    <arg name="input_control_cmd" default="/jetracer/cmd_drive"/>
    <arg name="jetracer_param"  default="$(find-pkg-share jetracer_node)/config/jetracer_node.param.yaml"/>
    <!-- JetRacer 制御ノードの起動 -->
    <node pkg="jetracer_node" 
          exec="jetracer_node" 
          name="jetracer_driver" 
          output="screen">

        <param from="$(var jetracer_param)"/>
        <remap from="/cmd_drive" to="$(var input_control_cmd)"/>
    </node>
    
</launch>
